### PR TITLE
Updated my README file with my blogs

### PR DESCRIPTION
**Pull request by FanyamTerkuma**

### CODE REVIEW

**Comment by FanyamTerkuma** [comment]
> Hello @FanyamTerkuma,
> 
> I've reviewed your recent PR for the README file, and it's looking great. However, in the `## Research` section, I recommend keeping a brief introduction and adding a link to your full blog post on accessibility instead of including the entire content within the `README`. This will maintain document clarity and offer readers easy access to the complete information. Keep up the good work!

Oh Ok @codewithmide , thanks for the feedback. :)

File: 2023/terkumafanyam/README.md [hunk 1]
```diff
@@ -1,12 +1,220 @@
...
+
+- [About Wagtail]( https://wagtail.org/about-wagtail/): I used this to get the general idea of Wagtail.
+- [web.dev](https://web.dev/accessibility/): This helped me to get familiar with accessibility.
+- [WebAIM](https://webaim.org/standards/wcag/checklist): Provided summaries of the [WCAG](https://www.w3.org/TR/WCAG22/) standards
+- [Silktide](https://www.youtube.com/playlist?list=PLWSYD-KxdDxsgf16rlLlotPHo2LLo0EGr): This playlist was immensely helpful. It has short and concise videos.
+
+
+## Recent Learnings (Blog post)
+
+### [Accessibility and Inclusivity in Web Development](https://terkumafanyam.hashnode.dev/accessibility-and-inclusivity-in-web-development)
```
Comment (thibaudcolas) [inline 2023/terkumafanyam/README.md:90]: Nice one! I think that’s a great post.

File: 2023/terkumafanyam/README.md [hunk 2]
```diff
@@ -1,12 +1,220 @@
...
+So far I have only talked about visual impairments but there are other forms of impairments. They can be physical, speech, cognitive, auditory or neurological impairments. As developers, we should aim to build websites or technologies that are accessible.
+
+Thanks for reading to the end. I hope you enjoyed your journey with me. :) <https://terkumafanyam.hashnode.dev/accessibility-and-inclusivity-in-web-development>
+
+
+### Checklist: Accessibility features documentation
+
+### Blog post on my knowledge of accessibility
+
+### [A Gentle Guide to Accessibility](https://terkumafanyam.hashnode.dev/a-gentle-introduction-to-accessibility)
```
Comment (thibaudcolas) [inline 2023/terkumafanyam/README.md:115]: This is good writing but seems to me like a very generic introductory post, which doesn’t do much to **demonstrate your expertise based on past experience**. The previous post was better in that respect.

File: 2023/terkumafanyam/README.md [hunk 3]
```diff
@@ -1,12 +1,220 @@
...
+#### Conclusion
+
+So far, we've looked at what web accessibility is and why it is important. We also looked at how we can test for web accessibility. Feel free to explore further on web accessibility. You can also look into other aspects of accessibility.
+
+Cheers for making it to the end of our journey. :)
+
+
+## Blog post on Technical writing
+
+## [A Beginner's Guide to React: Understanding its Power and Setting Up Your First App](https://terkumafanyam.hashnode.dev/a-beginners-guide-to-react-understanding-its-power-and-setting-up-your-first-app)
```
Comment (thibaudcolas) [inline 2023/terkumafanyam/README.md:150]: This seems to me like solid technical writing, but just one example. It’d have been better IMHO if you wrote about technical writing, so you could share multiple examples.

File: 2023/terkumafanyam/README.md (hunk 2 above)
Comment (FanyamTerkuma) [reply 2023/terkumafanyam/README.md:115]: Thank you for the feedback , I will work on another post that would be better in that regard. :)

File: 2023/terkumafanyam/README.md (hunk 3 above)
Comment (FanyamTerkuma) [reply 2023/terkumafanyam/README.md:150]: Thank you for the feedback. I will work on making a better post. :)

**Pull request merged**